scripts/data_overlap/get_group_overlap_stats.py

```diff
@@ -0,0 +1,91 @@
+import json
+import cattrs
+import os
+import argparse
+from typing import List, Dict
+
+from common.general import asdict_without_nones
+
+from data_overlap_spec import DataOverlapStats
+from light_scenario import GroupScenarioSpecs, GroupOverlapStats, AllGroupOverlapStats
+
+
+def save_group_overlap_stats_to_jsonl(all_group_overlap_stats: AllGroupOverlapStats, filename: str):
+    with open(filename, "a") as f:
+        f.write(json.dumps(asdict_without_nones(all_group_overlap_stats), ensure_ascii=False) + "\n")
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--overlap_stats_path", type=str, required=True, help="The path to the overlap stats file")
```

Comment:
use kebab case consistently everywhere

Just call it `--overlap-stats` (since path is explicit) or at least make it consistent with `--output-data` (which does not have `path`).